src/Analyzer.Cli.FunctionalTests/CommandLineParserTests.cs

```diff
@@ -92,7 +93,7 @@ public void AnalyzeDirectory_ValidInputValues_ReturnExpectedExitCode(bool useTes
         public void AnalyzeDirectory_DirectoryWithOtherJsonFiles_LogsExpectedErrorInSarif()
```

Comment:
Is this a good opportunity to update/remove this test, since the bug fix in #238 breaks the test?